CondFormats/HcalObjects/src/HcalInterpolatedTableFunctor.cc

```diff
@@ -0,0 +1,101 @@
+#include <algorithm>
+#include "FWCore/Utilities/interface/Exception.h"
+
+#include "CondFormats/HcalObjects/interface/HcalInterpolatedTableFunctor.h"
+
+inline static double interpolateStep(
```

Comment:
put this in an anonymous namespace or make less randomly reproducible name?
chances of collision are probably low though (also, one will need to compile with -O0 to make it a visible object)